exercises/09/before/ClientUI/Program.cs

```diff
@@ -96,7 +96,7 @@ static void RegisterBatchOfUsers()
             var random = new Random();
             var users = new[] { "Dennis", "Ramon", "Mauro", "Adam", "Udi", "David", "Szymon", "Tomasz", "Sean", "Kim" };
 
-            Parallel.For(0, 10, i => 
+            Parallel.For(0, 10, i =>
             {
                 RegisterNewUser(users[random.Next(9)]);

```

Comment:
Uff this is ugly. Uses Parrallel.For with an async body. Switch this to `Parallel.ForEachAsync` if you are targeting net6 or at least add some `GetAwaiter().GetResult()`?